ddl/ddl_tiflash_api.go

```diff
@@ -57,22 +57,156 @@ type TiFlashReplicaStatus struct {
 	IsPartition    bool
 }
 
+// TiFlashTick is type for backoff threshold.
+type TiFlashTick float64
+
+// PollTiFlashBackoffElement records backoff for each TiFlash Table.
+// `Counter` increases every `Tick`, if it reached `Threshold`, it will be reset to 0 while `Threshold` grows.
+// `TotalCounter` records total `Tick`s this element has since created.
```

Comment:
Why do I feel like these two fields are the same usage? [This](https://github.com/pingcap/tidb/pull/32317/files#diff-dca9a62898accc180490d462575cc0eae471c50e76c7348373075d5944addd93R134-R135) is the only place they do update it.